src/constants.ts

```diff
@@ -47,6 +47,10 @@ export const IS_MOBILE = (() => {
   return check;
 })();
 
+export const HAS_OFFSCREENCANVAS = Boolean((self as any).OffscreenCanvas);
+export const OFFSCREENCANVAS_SUPPORT_BITMAP = Boolean((self as any).OffscreenCanvas) && 
```

Comment:
Nit: Let's add another `_` between `OFFSCREEN` and `CANVAS` eg, `HAS_OFFSCREEN_CANVAS`